doc/manual/cookbooks/benchmarks/brittle_thrust_wedges/convergence_part.prm

```diff
@@ -0,0 +1,43 @@
+# Note that the Linear/Nonlinear solver tolerance should be sufficiently strict 
+# to avoid numerical instabilities.
+set Nonlinear solver scheme                 = single Advection, iterated Stokes
+set Nonlinear solver toleranc               = 1e-7
```

Comment:
this looks like it is incorrect (missing e). Can you check the rest of the file (by copying from the main prm maybe)?